client/src/css/Button.css

```diff
@@ -1,50 +1,57 @@
 .Button {
-  margin: 2rem 0 0 0;
-  padding: 1rem;
-  font-size: 1.1em;
-  border: 1px solid transparent;
-  background: transparent;
-  color: #636bb2;
+  margin: 1rem 0 0 0;
+  border: 0.0625rem solid transparent;
+  background-color: transparent;
+  color: #636BB2;
+  white-space: nowrap;
 }
 
 .Button:hover {
   color: #141523;
+  cursor: pointer;
 }
 
 .Button:focus {
   outline: 0;
 }
 
 .Button--background {
-  font-size: 1em;
-  color: #ffffff;
-  background: #595C7F;
-  border: 1px solid rgba(0, 0, 0, 0.04);
-  box-shadow: 0px -2px 0px 0px rgba(0, 0, 0, 0.25) inset,
-              0px 0px 1px 0px rgba(0, 0, 0, 0.25);
+  color: #FFF;
+  background-color: #595C7F;
+  border: 0.0625rem solid rgba(0, 0, 0, 0.04);
+  box-shadow: 0 -0.125rem 0 0 rgba(0, 0, 0, 0.25) inset,
+              0 0 0.0625rem 0 rgba(0, 0, 0, 0.25);
 }
 
 .Button--background:hover {
-  color: #ffffff;
-  background: #6d7194;
+  color: #FFF;
+  background-color: #6D7194;
 }
 
 .Button:disabled {
-  background: #71727d;
+  background-color: #71727D;
+  opacity: 0.5;
```

Comment:
Personally I feel opacity is underutilized. We have a white background, or at least somewhat white throughout the entire page. Reducing opacity is equivalent to lightening the background color, which is sort of what we want. 
It also stems from my aversion to throwing out what are essentially the magical numbers of hex-codes. It feels like we have countless shades of indigo, changing opacity very clearly defines that this is that one shade, just 50% brighter.